modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep_chunks/sweep_chunk.h

```diff
@@ -1,42 +1,127 @@
 #pragma once
 
-#include "framework/mesh/sweep_utilities/sweep_chunk_base.h"
+#include "modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep/angle_aggregation/angle_aggregation.h"
 #include "modules/linear_boltzmann_solvers/a_lbs_solver/lbs_structs.h"
+#include "modules/linear_boltzmann_solvers/a_lbs_solver/groupset/lbs_groupset.h"
+#include <functional>
 
 namespace opensn
 {
 namespace lbs
 {
 
-/**Base class for LBS sweepers*/
-class SweepChunk : public opensn::SweepChunk
+/**Sweep work function*/
+class SweepChunk
 {
 public:
-  SweepChunk(std::vector<double>& destination_phi,
-             std::vector<double>& destination_psi,
+  /**
+   * Convenient typdef for the moment call back function. See moment_callbacks.
+   *  Arguments are:
+   *  SweepChunk *
+   *  AngleSet *
```

Comment:
I get it that `SweepChunk` and `AngelSet` are descriptive enough, then do you want to keep these with no comments? Or do you think people would miss they are arguments (i.e. they would not be familiar with the C++ syntax) and this comment would help them?

Reply:
I left the description and removed the argument comment.